src/Controllers/BeheerEventAanmakenController.php

```
@@ -66,34 +70,65 @@ public function sendEvent(){
-            $this->render('beheer/event-aanmaken-stap-2', ['error' => 'Er is een fout opgetreden bij het aanmaken van het evenement.']);
+            $this->redirect('/beheer/event-aanmaken-stap-2');
         }
 
-    } 
-
+    }
+    
     public function sendEventStep2()
     {
-        $this->redirection('beheer/event');
+        $currentEventID = $_SESSION['register_data']['eventID'];
```

PHPMD (2.15.0)

Avoid unused local variables such as '$currentEventID'. (undefined)

(UnusedLocalVariable)